asset/elasticsearch_state_storage/interfaces.js

```diff
@@ -0,0 +1,8 @@
+
+// TODO: keeping for future use of typescript
```

Comment:
We shouldn't keep this in here because by the type we go to typescript this may be invalid.